README.md

```diff
@@ -6,5 +6,11 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails is a library of ruby code that is used as a frameowrk for building websites.
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
```

Comment:
The controller doesn't pass on the request to the model. Instead, it invokes a method on the model in order to accomplish whatever the request is asking it to do.